app/css/components/completed-heading.css

```
@@ -0,0 +1,21 @@
+.c-completed-heading {
```

```suggestion
.c-completed-exercise-heading {
```

Can you rename this in the various places pls (filename, etc)